mmv1/third_party/terraform/services/cloudidentity/resource_cloud_identity_group_sweeper.go.erb

```diff
@@ -1,5 +1,5 @@
 <% autogen_exception -%>
-package google
+package cloudidentity
```

Comment:
Should these be added to the test package instead of the resource level package?

Also have you tested that the sweepers clean up resources when relocated to these packages?

Reply:
The _sweeper.go files will be in the product package instead of product_test package. Otherwise, they cannot be imported into the file utils/gcp_sweeper_test.go.erb and initialized. 

I checked in the nightly tests, the sweeper `FirebaseAndroidApp` that was moved last Friday is still working.

I tested a couple of sweepers locally.

Reply:
-central1):
2023/06/26 15:58:28 [DEBUG] Running Sweeper (ApigeeKeystoresAliasesKeyCertFile) in region (us-central1)
2023/06/26 15:58:28 [INFO][SWEEPER_LOG] Starting sweeper for ApigeeKeystoresAliasesKeyCertFile
2023/06/26 15:58:28 [INFO] Authenticating using configured Google JSON 'credentials'...
2023/06/26 15:58:28 [INFO]   -- Scopes: [https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/userinfo.email]
2023/06/26 15:58:28 [INFO] Authenticating using configured Google JSON 'credentials'...
2023/06/26 15:58:28 [INFO]   -- Scopes: [https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/userinfo.email]
2023/06/26 15:58:28 [DEBUG] Waiting for state to become: [success]
2023/06/26 15:58:28 [DEBUG] Waiting for state to become: [success]
2023/06/26 15:58:28 [DEBUG] Retry Transport: starting RoundTrip retry loop
2023/06/26 15:58:28 [DEBUG] Retry Transport: request attempt 0
2023/06/26 15:58:28 [DEBUG] Retry Transport: Stopping retries, last request failed with non-retryable error: 
2023/06/26 15:58:28 [DEBUG] Completed Sweeper (ApigeeKeystoresAliasesKeyCertFile) in region (us-central1) in 454.310292ms
2023/06/26 15:58:28 Completed Sweepers for region (us-central1) in 454.490709ms
2023/06/26 15:58:28 Sweeper Tests for region (us-central1) ran successfully:
	- ApigeeKeystoresAliasesKeyCertFile
ok  	github.com/hashicorp/terraform-provider-google/google	1.229s